### ISSUE TITLE
RFC - GT++ fishing pond

### ISSUE THREAD

**Issue by sthegreat** [GTNewHorizons/NewHorizons#3377]
#### Which modpack version are you using?

2.0.4.6

#### What do you suggest instead/what changes do you propose?

Right now the GT++ Fishing Port seems to be decent as a means of biodiesel production. Nearly all parts of the multiblock can be made in mv age. However, the Inconel-792 ingots require a vacuum freezer, which gates the machine to hv instead. This makes it available a little bit too late to be used for biodiesel production, since by hv most people intending to use biodiesel have likely already done so by another route and are now thinking about going to the moon for nuclear as their next power upgrade.

Should the freezing requirement be removed? Doing so would add another method of obtaining fuels in the mv age. However, we must make sure this machine is not too powerful so it does not make other seed/fish oil production setups obsolete. Perhaps some sort of running cost could be added to it, like fish bait to make it operate at full speed?

**Comment by Wolozo7** [comment]
I don't necessarily disagree, but always thought the FIsh Catchers to be supposed to be the early game source of fish, while the Fishing Pond was more of a later source (although i agree that there isn't much need for fish later on).

**Comment by draknyte1** [comment]
Wolozo is correct and I won't change this. If used correctly Fishing Ports can net positive power generation. (Running a heap in parallel at LV)

You don't *need* a Vacuum freezer either, just an Alloy Blast Smelter.. which requires Ti, which doesn't require the moon if you're keen to rush Rutile processing. (Using TC for duping materials, you can do this all very early)

**Comment by Anarack** [comment]
Yes you do need a vac freezer to do rutile based titanium extraction, it needs kanthal coils, and getting the staballoy for a alloy blast furnace and that needs a vac freezer or a alloy blast furnace anyway in gt:nh so the fishing pond is still behind a vac freezer.

**Comment by draknyte1** [comment]
Does staballoy produce hot ingots? I'll change that if you drop me an issue.

**Comment by Anarack** [comment]
Yep, i'll dump it on your github.

**Comment by sthegreat** [comment]
Additionally, constructing GT:NH's alloy blast smelter needs tungstensteel, tungsten metal, and also synthetic rubber due to the IV alloy smelter in the recipe, so you can't exactly skip the vacuum freezer requirement...
